tema3/src/com/sci/bv/Main.java

```diff
@@ -0,0 +1,47 @@
+package com.sci.bv;
+
+import java.util.Scanner;
+/**
+ * a program that reads three numbers from the console and prints the biggest of them.
+ * @author Iagar Denisa
+ */
+public class Main {
+    /**
+     *  Use the getMax() method and prints
+     *  the biggest number.
+     *   @param args unused.
+     */
+
+    public static void main(String[] args) {
+        System.out.println("Read the first number: ");
+
+        Scanner input = new Scanner(System.in);
+        int firstNum = input.nextInt();
+
+        System.out.println(" Read the second number: ");
+        int secondNum = input.nextInt();
+
+        System.out.println("Read the third number: ");
+        int thirdNum = input.nextInt();
+
+        int max = Math.max(getMax(firstNum,secondNum),thirdNum);
```

Comment:
why use Math.max instead of getMax twice?
getMax(getMax(firstNum, secondNum), thirdNum)